feedback/lib/src/better_feedback.dart

```diff
@@ -18,7 +18,49 @@ typedef OnSubmit = void Function(
 
 /// A function that returns a Widget that prompts the user for feedback and
 /// calls [OnSubmit] when the user wants to submit their feedback.
-typedef FeedbackBuilder = Widget Function(BuildContext, OnSubmit);
+///
+/// A non-null controller is provided if the sheet is set to draggable in the
+/// feedback theme.
+/// If the sheet is draggable, the non null controller should be passed into a
+/// scrollable widget to make the feedback sheet expand when the widget is
+/// scrolled. Typically, this will be a `ListView` or `SingleChildScrollView`
+/// wrapping the feedback sheet's content.
+/// See: [FeedbackThemeData.sheetIsDraggable] and [DraggableScrollableSheet].
+typedef FeedbackSheetBuilder = Widget Function(
+  BuildContext,
+  OnSubmit,
+  ScrollController?,
+);
+
+/// A drag handle to be placed at the top of a draggable feedback sheet.
+///
+/// This is a purely visual element that communicates to users that the sheet
+/// can be dragged to expand it.
+///
+/// It should be placed in a stack over the sheet's scrollable element so that
+/// users can click and drag on it-the drag handle ignores pointers so the drag
+/// will pass through to the scrollable beneath.
+///
+/// TODO(caseycrogers): Replace this with a pre-built drag handle above the
+///   builder function once `DraggableScrollableController` is available in
+///   production.
+///   See: https://github.com/flutter/flutter/pull/92440.
```

Comment:
This shouldn't be a part of the doc comments.
```suggestion
// TODO(caseycrogers): Replace this with a pre-built drag handle above the
//   builder function once `DraggableScrollableController` is available in
//   production.
//   See: https://github.com/flutter/flutter/pull/92440.
```
Maybe we can already create an issue for that, so we don't forget.

Reply:
https://github.com/ueman/feedback/issues/150